Sources/LiveViewNative/Modifiers/Text Input and Output Modifiers/KeyboardShortcutModifier.swift

```diff
@@ -0,0 +1,156 @@
+//
+//  KeyboardShortcutModifier.swift
+//  
+//
+//  Created by murtza on 18/05/2023.
+//
+
+import SwiftUI
+
+/// Assigns a keyboard shortcut to the modified control.
+///
+/// ```html
+/// <Button modifiers={keyboard_shortcut(@native, key: :"+")}>Click Me!</Button>
+/// <Button modifiers={keyboard_shortcut(@native, key: :"7")}>Click Me!</Button>
+/// <Button modifiers={keyboard_shortcut(@native, key: :s, event_modifiers: [:command])}>Click Me!</Button>
+/// <Button modifiers={keyboard_shortcut(@native, key: :s, event_modifiers: [:command, :shift])}>Click Me!</Button>
+/// ```
+///
+/// ## Arguments
+/// * ``key``
+/// * ``eventModifiers``
+#if swift(>=5.8)
+@_documentation(visibility: public)
+#endif
+@available(iOS 14.0, macOS 11.0, *)
+struct KeyboardShortcutModifier: ViewModifier, Decodable {
+    #if !os(iOS) && !os(macOS)
+    typealias KeyEquivalent = Never
+    #endif
+    /// The key equivalent that the user presses in conjunction with any specified modifier keys to activate the shortcut.
+    /// One of the `KeyEquivalent` enumerations.
+    ///
+    /// Possible values:
+    /// * `up_arrow`
+    /// * `down_arrow`
+    /// * `left_arrow`
+    /// * `right_arrow`
+    /// * `clear`
+    /// * `delete`
+    /// * `end`
+    /// * `escape`
+    /// * `home`
+    /// * `page_up`
+    /// * `page_down`
+    /// * `return`
+    /// * `space`
+    /// * `tab`
+    /// * `a-z`
+    /// * `A-Z`
+    /// * `0-9`
+    /// * `!@#$%^*()-_=+[]{}|:"'<>,./?`
+    #if swift(>=5.8)
+    @_documentation(visibility: public)
+    #endif
+    private var key: KeyEquivalent
+    /// The modifier keys that the user presses in conjunction with a key equivalent to activate the shortcut.
+    /// One of the `EventModifiers` enumerations.
+    ///
+    /// Possible values:
+    /// * `all`
+    /// * `caps_lock`
+    /// * `command`
+    /// * `control`
+    /// * `numeric_pad`
+    /// * `option`
+    /// * `shift`
+    #if swift(>=5.8)
+    @_documentation(visibility: public)
+    #endif
+    private var eventModifiers: EventModifiers = []
```

Comment:
There's some naming mismatches in this implementation. Some places have `event_modifiers` and others have `event_modifier`.

I think we should stick with SwiftUI's naming and call it just `modifiers`.